add streetscience post (#232)
usegalaxy-eu/website: _posts/2019-04-04-streetscience-first-public-workshop.md

---
site: freiburg
title: 'BeerDEcoded: 1st public workshop'
external: https://streetscience.community/events/2019-03-10-first_public_workshop/
tags: [streetscience]
---

> On the 31st of March, we hold our first public workshop for the BeerDEcoded project in Freiburg.

Reviewer: ```suggestion
On the 31st of March, we held our first public workshop for the BeerDEcoded project in Freiburg.
```

> Some students and external people participated to this event: 16 peoples in total, including us.

Reviewer: ```suggestion
Several students and external people participated in this event, making a total of 16 people, including us.
```
